tests/installation/ipxe_install.pm

```diff
@@ -290,6 +346,13 @@ sub run {
         return;
     }
 
+    if (is_agama) {
+        check_screen([qw(load-linux-kernel load-initrd)], 240);
+        record_info("Installing", "Please check the expected product is being installed");
+        assert_screen('agama-installer-live-root', 400);
+        return;
+    }
```

Comment:
I think it would be better only at most asserting kernel/initird. Anything following should belong to agama module.

Reply:
Yup, let me do that